Add missing entries in CHANGELOG.adoc (#39)
asciidoctor/asciidoctor-chart: CHANGELOG.adoc

== Unreleased

> ## [1.0.0] - 2022-10-07
> ### Added

Reviewer: ```suggestion
```

=== icon:rocket[] Added

* Introduce a changelog (#30)
* Added support for using the JavaScript libraries offline (#25)
* Add captioned title on charts (#18)
* Improved documentation
* Added attributes to define chart resources directory (#26)
* Introduce visual regression tests (#34)

> === icon:heartbeat[] Changed

Reviewer: we should include https://github.com/asciidoctor/asciidoctor-chart/pull/17

Author: Split it between Changed and Fixed cause it was both bugfixing and general changes

* Use 0.11.x as default version for Chartist.js instead of latest (#28)
* Update c3.js defaults to latest version 0.7.20 (#32) 

> === icon:wrench[] Fixed

Reviewer: We can add this bug fix: https://github.com/asciidoctor/asciidoctor-chart/pull/13

Author: Done.

> === icon:wrench[] Fixed

Reviewer: We should include https://github.com/asciidoctor/asciidoctor-chart/pull/24
It includes assets only when a chart engine is used (previously all assets were included for all engines)

Author: Done

> * Fixed ")" in top left corner

Reviewer: We can remove this fix since it wasn't part of any release

Author: Removed it.

== Unreleased

=== icon:rocket[] Added

* Introduce a changelog (#30)

> * Added support for using the JavaScript libraries offline (#25)

Reviewer: ```suggestion
* Add support for using the JavaScript libraries offline (#25)
```

* Add captioned title on charts (#18)
* Improved documentation

> * Added attributes to define chart resources directory (#26)

Reviewer: ```suggestion
* Add attributes to define chart resources directory (#26)
```

* Introduce visual regression tests (#34)

=== icon:heartbeat[] Changed

* Use 0.11.x as default version for Chartist.js instead of latest (#28)
* Update c3.js defaults to latest version 0.7.20 (#32) 
* Upgrade Chart.js from 1.0.2 to 3.7.0 and update code accordingly (#17)
* Use https protocol (instead of http) when loading scripts from cdnjs.cloudflare.com (#17)
* Apply RuboCop to asciidoctor-chart.gemspec (#17)

> * Add rubygems_mfa_required in gemspec (#17)

Reviewer: ```suggestion
* Enable multi-factor authentication when publishing to https://rubygems.org/ (#17)
```

=== icon:wrench[] Fixed

> * Fixed issue (#12) that we can only have one chartjs graph per page with (#13)

Reviewer: ```suggestion
* Fix an issue to allow more than one Chart.js graphs per page (#13)
```

* Include assets (styles, scripts)  when an chart engine is used (#24)

> * Fixed issue with chart.js graph overlaping text when width is specified (#8) with (#17)

Reviewer: ```suggestion
* Fix an issue with Chart.js where text was overlapping the graph when width was specified (#17)
```
